src/platform/plugins/shared/dashboard/docs/openapi/entrypoint.yaml

```diff
@@ -0,0 +1,2658 @@
+                                description: The title of the section.
+                                type: string
+                            required:
+                              - title
+                              - gridData
+                      type: array
+                    refreshInterval:
+                      additionalProperties: false
+                      description: A container for various refresh interval settings
+                      type: object
+                      properties:
+                        display:
+                          deprecated: true
+                          description: A human-readable string indicating the refresh frequency. No longer used.
+                          type: string
+                        pause:
+                          description: Whether the refresh interval is set to be paused while viewing the dashboard.
+                          type: boolean
+                        section:
+                          deprecated: true
+                          description: No longer used.
+                          type: number
+                        value:
+                          description: A numeric value indicating refresh frequency in milliseconds.
+                          type: number
+                      required:
+                        - pause
+                        - value
+                    tags:
+                      items:
+                        description: An array of tags applied to this dashboard
+                        type: string
+                      type: array
+                    timeFrom:
+                      description: An ISO string indicating when to restore time from
+                      type: string
+                    timeRestore:
+                      default: false
+                      description: Whether to restore time upon viewing this dashboard
+                      type: boolean
+                    timeTo:
+                      description: An ISO string indicating when to restore time from
+                      type: string
+                    title:
+                      description: A human-readable title for the dashboard
+                      type: string
+                    version:
+                      deprecated: true
+                      type: number
+                  required:
+                    - title
+                    - options
+                references:
+                  items:
+                    additionalProperties: false
+                    type: object
+                    properties:
+                      id:
+                        type: string
+                      name:
+                        type: string
+                      type:
+                        type: string
+                    required:
+                      - name
+                      - type
+                      - id
+                  type: array
+                spaces:
```

Comment:
It looks like this is called `namespaces` in the other models? Can we make the field names consistent?

Reply:
IMO, `spaces` would be the preferred term across all APIs as that is the term used in Kibana UI. But I also know that this lacks consistency with other APIs. 

A few existing examples of this discrepancy in different APIs.

- [Data views](https://www.elastic.co/docs/api/doc/kibana/operation/operation-createdataviewdefaultw#operation-createdataviewdefaultw-body-application-json-data_view-namespaces) uses `namespaces`
- [Synthetics monitors](https://www.elastic.co/docs/api/doc/kibana/operation/operation-post-synthetic-monitors#operation-post-synthetic-monitors-body-application-json-namespace) uses `namespace`
- [Roles](https://www.elastic.co/docs/api/doc/kibana/operation/operation-post-security-roles#operation-post-security-roles-body-application-json-roles-kibana-spaces) uses `spaces`

Reply:
`namespaces`, `namespace`, and `spaces` might sound like the same but they're significantly different i.t.o raw ES document attributes. `namespaces` and `namespace` relate to if the saved object can be shared or not whereas `spaces` should be interpreted as the array of spaces in which the object is available.
I recommend we stay consistent with the document representation and use `namespaces`.